plugins/tekton/README.md

```diff
@@ -172,14 +172,20 @@ Tekton is a front-end plugin that enables you to view the `PipelineRun` resource
 
 1. Go to the **CI/CD** tab.
 
-   The **CI/CD** tab displays the latest `PipelineRun` resources associated to a Kubernetes cluster. The resources include tasks to complete. When you hover the mouse pointer on a task card, you can view the steps to complete that particular task.
+   The **CI/CD** tab displays the latest `PipelineRun` resources associated with a Kubernetes cluster. The resources include tasks to complete. When you hover the mouse pointer on a task card, you can view the steps to complete that particular task.
 
    ![ci-cd-tab-tekton](./images/tekton-plugin-user1.png)
 
    There is also a **GO TO TEKTON** option at the bottom, which redirects you to the **TEKTON** tab.
 
-1. Click **GO TO TEKTON** or select the **TEKTON** tab in the entity view page.
+1. Click **GO TO TEKTON** or select the **TEKTON** tab on the entity view page.
 
    The **TEKTON** tab contains the list of pipeline runs related to a cluster. The list contains pipeline run details, such as **NAME**, **STATUS**, **TASK STATUS**, **STARTED**, and **DURATION**.
 
    ![tekton-tab](./images/tekton-plugin-user2.png)
+
+1. Select a PipelineRun to view the related resources with a Kubernetes cluster.
```

Comment:
```suggestion
2. Click on a PipelineRun name to view the PipelineRun visualization.
```

Reply:
Done, changed the sentence as follows:

Click a PipelineRun name in the list to view the PipelineRun visualization.